### PR TITLE
[Automated] Update actions

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
```shell
# meta: knative/.github
# main: knative/docs
cp $(find "${GITHUB_WORKSPACE}/meta/workflow-templates" -type f -name '*.yaml') \
  "${GITHUB_WORKSPACE}/main/.github/workflows/"
```
/cc abrennan89 dprotaso vaikas
/assign abrennan89 dprotaso vaikas

### CODE REVIEW

**Comment by dprotaso** [comment]
@mattmoor no update codegen in the docs repo

**Comment by mattmoor** [comment]
Yeah, my version of these has existence checks on `hack/update-codegen.sh` for repos like this.

We should fix this upstream.

**Comment by mattmoor** [comment]
This should fix it: https://github.com/knative-sandbox/.github/pull/42

File: .github/workflows/knative-verify.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - name: Check out code onto GOPATH
+      uses: actions/checkout@v2
+      with:
+        path: ./src/knative.dev/${{ github.event.repository.name }}
+
+    - name: Update Codegen
+      shell: bash
+      run: |
+        pushd ./src/knative.dev/${{ github.event.repository.name }}
+        ./hack/update-codegen.sh
```
Comment (dprotaso) [inline .github/workflows/knative-verify.yaml:60]: looks like the change wasn't pulled in?

Comment (mattmoor) [reply .github/workflows/knative-verify.yaml:60]: hasn't run yet 😉

Comment (mattmoor) [reply .github/workflows/knative-verify.yaml]: has now :D

**Pull request merged**